### PR TITLE
Upgrade to .NET Framework 4.6 to fix TLS handshake with GitHub

### PR DESCRIPTION
**Pull request by bradchristensen**

### CODE REVIEW

File: LIC.Malone.Client.Desktop/LIC.Malone.Client.Desktop.csproj [hunk 1]
```diff
@@ -9,12 +9,28 @@
...
+    <InstallFrom>Disk</InstallFrom>
+    <UpdateEnabled>false</UpdateEnabled>
+    <UpdateMode>Foreground</UpdateMode>
+    <UpdateInterval>7</UpdateInterval>
+    <UpdateIntervalUnits>Days</UpdateIntervalUnits>
+    <UpdatePeriodically>false</UpdatePeriodically>
+    <UpdateRequired>false</UpdateRequired>
+    <MapFileExtensions>true</MapFileExtensions>
+    <ApplicationRevision>0</ApplicationRevision>
+    <ApplicationVersion>1.0.0.%2a</ApplicationVersion>
```
Comment (stajs) [inline LIC.Malone.Client.Desktop/LIC.Malone.Client.Desktop.csproj]: This looks a bit weird. I'm sure it's fine?

File: LIC.Malone.Client.Desktop/LIC.Malone.Client.Desktop.csproj [hunk 2]
```diff
@@ -256,6 +283,18 @@
   <ItemGroup>
     <Content Include="Resources\Malone.svg" />
   </ItemGroup>
+  <ItemGroup>
+    <BootstrapperPackage Include=".NETFramework,Version=v4.6.1">
+      <Visible>False</Visible>
+      <ProductName>Microsoft .NET Framework 4.6.1 %28x86 and x64%29</ProductName>
+      <Install>true</Install>
+    </BootstrapperPackage>
+    <BootstrapperPackage Include="Microsoft.Net.Framework.3.5.SP1">
```
Comment (stajs) [inline LIC.Malone.Client.Desktop/LIC.Malone.Client.Desktop.csproj]: Curious that it's calling out .NET 3.5 (again, spose it's fine?).

**Comment by stajs** [comment]
I couldn't test the upgrade from v0.9.6 to v0.9.7 (obviously!), it's a shame that got broke. Did you consider wrapping the update check in a try/catch for future crypto deprecations?

I didn't get a warning on installing v0.9.6, but I did for v0.9.7: 

image

Should think about signing the installer to avoid that.

File: LIC.Malone.Client.Desktop/LIC.Malone.Client.Desktop.csproj (hunk 1 above)
Comment (bradchristensen) [reply LIC.Malone.Client.Desktop/LIC.Malone.Client.Desktop.csproj]: "%2a" looks to be the URL-encoded form of "*" 😛

Other than that, I'm not actually sure where this came from. There are no similar changes in the Core csproj... could be related to the .NET upgrade, maybe specific to the project-type? I presume it's fine?

Comment (stajs) [reply LIC.Malone.Client.Desktop/LIC.Malone.Client.Desktop.csproj]: Yeah, I think so. It builds right!

**Pull request merged**